### PR TITLE
Updated research.rst

### PR DESCRIPTION
**Pull request by anranjiao**

### CODE REVIEW

**Comment by lululxvi** [comment]
Please resolve the conflicts.

File: docs/user/research.rst [hunk 1]
```diff
@@ -3,168 +3,189 @@ Research
 
 DeepXDE has been used in
 
-- > 190 universities, e.g.,
+- > 180 universities, e.g.,
```
Comment (lululxvi) [inline docs/user/research.rst:6]: Is this number correct? It was 190.

File: docs/user/research.rst [hunk 2]
```diff
@@ -3,168 +3,189 @@ Research
...
   `University of Manchester <https://www.manchester.ac.uk>`_,
+  `University of Chinese Academy of Sciences <https://english.ucas.ac.cn>`_,
+  `Georgia Institute of Technology <https://www.gatech.edu>`_,
+  `Boston University <https://www.bu.edu>`_,
+  `University of Southern California <https://www.usc.edu>`_,
   `University of Wisconsin Madison <https://www.wisc.edu>`_,
+  `Technical University of Munich <https://www.tum.de/en>`_,
   `University of California, Santa Barbara <https://www.ucsb.edu>`_,
-  `Boston University <https://www.bu.edu>`_,
+  `University of Birmingham <https://www.birmingham.ac.uk/index.aspx>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:41]: Delete `index.aspx`

File: docs/user/research.rst [hunk 3]
```diff
@@ -3,168 +3,189 @@ Research
...
-  `South China Normal University <http://english.scnu.edu.cn>`_,
+  `Dortmund University of Technology <https://www.tu-dortmund.de/en>`_,
+  `University of Los Andes <https://uniandes.edu.co/en>`_,
+  `University of Stuttgart <https://www.uni-stuttgart.de/en>`_,
   `Universidad de salamanca <https://www.usal.es/universidad-de-salamanca>`_,
   `Harbin Engineering University <https://english.hrbeu.edu.cn>`_,
-  `Graz University of Technology <https://www.tugraz.at/en/home>`_,
-  `Missouri University of Science and Technology <https://www.mst.edu>`_,
-  `California Polytechnic State University <https://www.calpoly.edu>`_,
+  `Universiti Teknologi Petronas <https://www.utp.edu.my/Pages/Home.aspx>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:141]: Delete `Pages/Home.aspx`

File: docs/user/research.rst (hunk 1 above)
Comment (anranjiao) [reply docs/user/research.rst:6]: It was 170, and I think it was changed to 190 in another pull request. It should be 180

File: docs/user/research.rst [hunk 4]
```diff
@@ -3,168 +3,189 @@ Research
...
-  `South China Normal University <http://english.scnu.edu.cn>`_,
+  `Dortmund University of Technology <https://www.tu-dortmund.de/en>`_,
+  `University of Los Andes <https://uniandes.edu.co/en>`_,
+  `University of Stuttgart <https://www.uni-stuttgart.de/en>`_,
   `Universidad de salamanca <https://www.usal.es/universidad-de-salamanca>`_,
   `Harbin Engineering University <https://english.hrbeu.edu.cn>`_,
-  `Graz University of Technology <https://www.tugraz.at/en/home>`_,
-  `Missouri University of Science and Technology <https://www.mst.edu>`_,
-  `California Polytechnic State University <https://www.calpoly.edu>`_,
+  `Universiti Teknologi Petronas <https://www.utp.edu.my/Pages>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:141]: Delete `/Pages`

File: docs/user/research.rst [hunk 5]
```diff
@@ -205,20 +231,58 @@ DeepXDE has been used in
...
+#. H\. Nganguia & D. Palaniappan. `Ciliary propulsion through non-uniform flows <https://www.cambridge.org/core/journals/journal-of-fluid-mechanics/article/ciliary-propulsion-through-nonuniform-flows/124528EA9A3D0C1E4FC14A4B185FB949>`_. *Journal of Fluid Mechanics*, 986:A14, 2024.
+#. A\. T. Deresse & T. T. Dufera. `Exploring physics-informed neural networks for the generalized nonlinear Sine-Gordon equation <https://onlinelibrary.wiley.com/doi/full/10.1155/2024/3328977>`_. *Applied Computational Intelligence and Soft Computing*, 2024(1):3328977, 2024.
+#. H\. Qiumei, M. Jiaxuan, & X. Zhen. `Mass-preserving spatio-temporal adaptive PINN for Cahn-Hilliard equations with strong nonlinearity and singularity <https://arxiv.org/pdf/2404.18054>`_, 2024.
+#. Z.\ Zhang, J.-H. Lee, L. Sun, & G. X. Gu. `Weak-formulated physics-informed modeling and optimization for heterogeneous digital materials <https://academic.oup.com/pnasnexus/article/3/5/pgae186/7667149>`_. *J PNAS Nexus*, 3(5):pgae186, May 2024.
```
Comment (lululxvi) [inline docs/user/research.rst:261]: `PNAS Nexus` --- no `J`.

File: docs/user/research.rst [hunk 6]
```diff
@@ -401,7 +464,19 @@ PINN
 DeepONet
```
Comment (lululxvi) [inline docs/user/research.rst:464]: Change to `Deep neural operators`

File: docs/user/research.rst [hunk 7]
```diff
@@ -443,4 +518,4 @@ Multi-fidelity NN
 Others
 ------
 
-#. A\. Jiao, H. He, R. Ranade, J. Pathak, & L. Lu. `One-shot learning for solution operators of partial differential equations <https://arxiv.org/abs/2104.05512>`_. *arXiv preprint arXiv:2104.05512*, 2021.
+#. A\. Jiao, H. He, R. Ranade, J. Pathak, & L. Lu. `One-shot learning for solution operators of partial differential equations <https://arxiv.org/abs/2104.05512>`_. *arXiv preprint arXiv:2104.05512*, 2024.
```
Comment (lululxvi) [inline docs/user/research.rst:521]: Move this under neural operators

File: docs/user/research.rst [hunk 8]
```diff
@@ -442,5 +518,3 @@ Multi-fidelity NN
 
 Others
```
Comment (lululxvi) [inline docs/user/research.rst:519]: Delete `Others`

File: docs/user/research.rst [hunk 9]
```diff
@@ -3,168 +3,189 @@ Research
...
-  `University of Los Andes <https://uniandes.edu.co/en>`_,
-  `Kuwait University <https://kuweb.ku.edu.kw>`_,
   `University of Engineering and Management <https://uem.edu.in>`_,
-  `Boise State University <https://www.boisestate.edu>`_,
-  `International School for Advanced Studies <https://www.sissa.it>`_,
-  `University of Bayreuth <https://www.uni-bayreuth.de/en>`_,
-  `ITMO University <https://en.itmo.ru>`_,
-  `University of Calcutta <https://www.caluniv.ac.in>`_
-- > 30 national labs and research institutes, e.g.,
+- > 35 national labs and research institutes, e.g.,
```
Comment (lululxvi) [inline docs/user/research.rst:188]: `> 30`

File: docs/user/research.rst [hunk 10]
```diff
@@ -205,20 +231,58 @@ DeepXDE has been used in
   `Bosch <https://www.bosch.us>`_,
   `ExxonMobil <https://corporate.exxonmobil.com>`_,
   `General Motors <https://www.gm.com>`_,
+  `Intel Corporation <https://www.intel.com>`_,
   `RocketML <https://www.rocketml.net>`_,
   `Saudi Aramco <https://www.aramco.com>`_,
   `Shell <https://www.shell.com>`_,
   `SoftServe <https://www.softserveinc.com/en-us>`_,
-  `Fraunhofer Institute for Integrated Systems and Device Technology IISB <https://www.iisb.fraunhofer.de>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:212]: Why is this deleted?

Comment (anranjiao) [reply docs/user/research.rst:212]: Moved it to research institute
